packages/components/autocomplete/src/use-autocomplete.tsx

```diff
@@ -473,11 +474,7 @@ export const useAutocomplete = <T extends string | string[] = string>(
           const { label, items = [], ...props } = item
 
           return (
-            <AutocompleteOptionGroup
-              key={i}
-              label={label as string}
-              {...(props as HTMLUIProps<"ul">)}
-            >
+            <AutocompleteOptionGroup key={i} label={label as string} {...props}>
```

Comment:
This fix is no longer needed in #2803. Please remove this modification.

Reply:
fixed